src/lib/stitching/causality/stitching.ts

```diff
@@ -1,5 +1,25 @@
 import gql from "lib/gql"
+import { formatMoney } from "accounting"
 import { GraphQLSchema } from "graphql"
+import {
+  moneyMajorResolver,
+  symbolFromCurrencyCode,
+} from "schema/v2/fields/money"
+
+const resolveLotCentsFieldToMoney = (centsField) => {
```

Comment:
Extract the resolver to a function. I'm still interested in a better way to be able to automatically resolve to the `Money` type. Since that's more implementation details, I'll leave it for a separate PR.